api/app/config.py

```diff
@@ -24,12 +24,12 @@ class BaseConfig:
         f"{MY_SOLID_APP_DB_NAME}"
     )
 
-    MAIL_SERVER = os.environ.get("MY_SOLID_APP_MAIL_SERVER", "stmp.server.com")
-    MAIL_PORT = int(os.environ.get("MY_SOLID_APP_MAIL_PORT", 465))
+    MAIL_SERVER = os.environ.get("MY_SOLID_APP_MAIL_SERVER", "localhost")
+    MAIL_PORT = int(os.environ.get("MY_SOLID_APP_MAIL_PORT", 1025))
     MAIL_USE_TLS = False
-    MAIL_USE_SSL = True
```

Comment:
Can you put this as an environment variable? In production it needs to be set at `True`, so it would be nice to have it configurable. We can do it `False` as default, en when deploying set it to `True`?